cmd/ssh.go

```diff
@@ -192,16 +226,78 @@ func (cmd *SSHCmd) jumpContainer(ctx context.Context, devPodConfig *config.Confi
 	}
 
 	// tunnel to container
-	return tunnel.NewContainerTunnel(client, cmd.Proxy, log).Run(ctx, func(ctx context.Context, containerClient *ssh.Client) error {
-		// we have a connection to the container, make sure others can connect as well
-		unlockOnce.Do(client.Unlock)
+	return tunnel.NewContainerTunnel(client, cmd.Proxy, log).
+		Run(ctx, func(ctx context.Context, containerClient *ssh.Client) error {
+			// we have a connection to the container, make sure others can connect as well
+			unlockOnce.Do(client.Unlock)
+
+			// start ssh tunnel
+			return cmd.startTunnel(
+				ctx,
+				devPodConfig,
+				containerClient,
+				client.WorkspaceConfig().IDE.Name,
+				log,
+			)
+		})
+}
+
+func (cmd *SSHCmd) reverseForwardPorts(
```

Comment:
this could potentially be invoked twice with similar arguments, is it safe to do so?

Reply:
What do you mean? as in "run multiple times the gpg-forwarding"?

Reply:
nope, but if you provide the flag **and** gpg forwarding, this function will be called twice (or I missed smth)

Reply:
Tested and it indeed created problems (sometimes)
Fixed it now thanks!